### PR TITLE
refactor(dockerfile): Remove stackrox-data Dockerfile

### PR DESCRIPTION
**Pull request by connorgorman**
## Description

stackrox-data dockerfile is unnecessary and makes our builds more complicated. Unify it as a part of the multi-stage build. Also removing ldb is fine at this point as it is a rocksdb utility which we are moving away from

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

CI + API Docs and network entities showed up in network graph

### CODE REVIEW

**Review by gavin-stackrox — APPROVED** [review]

**Review by msugakov — APPROVED** [review]
Please address two new things before merging.

File: image/rhel/create-bundle.sh [hunk 1]
```diff
@@ -9,48 +9,19 @@ die() {
...
-}
-
-INPUT_ROOT="$1"
-DATA_IMAGE="$2"
-BUILDER_IMAGE="$3"
-OUTPUT_DIR="$4"
-# Install the PG repo natively if true (versus using a container)
-NATIVE_PG_INSTALL="${5:-false}"
+INPUT_ROOT="${1:-}"
+BUILDER_IMAGE="${2:-}"
```
[the remark this answers is not in the document]
Comment (connorgorman) [reply image/rhel/create-bundle.sh:13]: Good find, thanks

File: image/rhel/fetch-stackrox-data.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Fetches data used by the stackrox:main image
+
+set -euox pipefail
```
[the remark this answers is not in the document]
Comment (connorgorman) [reply image/rhel/fetch-stackrox-data.sh:5]: will swap

**Pull request merged**